ml_metadata/tools/mlmd_bench/fill_context_edges_workload_test.cc

```diff
@@ -0,0 +1,295 @@
+/* Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+#include "ml_metadata/tools/mlmd_bench/fill_context_edges_workload.h"
+
+#include <gtest/gtest.h>
+#include "absl/memory/memory.h"
+#include "ml_metadata/metadata_store/metadata_store.h"
+#include "ml_metadata/metadata_store/metadata_store_factory.h"
+#include "ml_metadata/metadata_store/test_util.h"
+#include "ml_metadata/proto/metadata_store.pb.h"
+#include "ml_metadata/proto/metadata_store_service.pb.h"
+#include "ml_metadata/tools/mlmd_bench/proto/mlmd_bench.pb.h"
+#include "ml_metadata/tools/mlmd_bench/util.h"
+#include "tensorflow/core/lib/core/status_test_util.h"
+
+namespace ml_metadata {
+namespace {
+
+constexpr int kNumberOfOperations = 100;
+constexpr int kNumberOfExistedTypesInDb = 132;
+constexpr int kNumberOfExistedNodesInDb = 145;
+constexpr int kNumberOfExistedContextEdgesInDb = 105;
+constexpr int kNumberOfEdgesPerRequest = 5;
+
+constexpr auto config_str = R"(
```

Comment:
drop auto, use types

Reply:
Got you. As for the `auto` in fill_nodes_workload_test.cc and fill_types_workload_test.cc, will change them accordingly along with the later TODOs in order to decrease the number of changed files in this PR.